Scripts/script-AddEvidence.yml

```diff
@@ -17,11 +17,9 @@ script: |-
       if (isValidRes(entries)) {
           for (var j=0;j<entries.length;j++){
               var ent=entries[j];
-              if (((ent.ContentsFormat == 'table' || ent.ContentsFormat == 'markdown') && ent.Contents) || ent.Type == entryTypes['file']) {
-                  var res = executeCommand('markAsEvidence', {'id': entryID, 'description': desc});
-                  if (!isValidRes(res)) {
-                      return res;
-                  }
+              var res = executeCommand('markAsEvidence', {'id': entryID, 'description': desc});
```

Comment:
I think the idea here was to ignore simple text responses like `yes` or `no` which might be part of the array of last entries